src/SFML/Window/Android/VideoModeImpl.cpp

```diff
@@ -31,19 +31,17 @@
 #include <SFML/Window/VideoModeImpl.hpp>
 
 #include <mutex>
+#include "SFML/Window/VideoMode.hpp"
```

Comment:
```suggestion
```
Looks like `clangd` or something similar automatically added this. This happens to me sometimes too.